src/main/java/org/jabref/logic/util/URLUtil.java

```diff
@@ -82,25 +83,37 @@ public static String cleanGoogleSearchURL(String url) {
      * @return true if <c>url</c> contains a valid URL
      */
     public static boolean isURL(String url) {
+        if (!url.matches("^[a-zA-Z]+://.*")) {
```

Comment:
Please refine the JavaDoc. It should state that an URL is requried to have a scheme and an authority.

See https://en.wikipedia.org/wiki/URL#Syntax for details.

Normally an URL has an "authority" part - which is IMOH requried at HTTP(S) URLs.

Reply:
Use a pre-compiled pattern with `private static final Pattern SCHEME_PREFIX ...`